cc/tflite_micro/apps/hello_world/main.cc

```diff
@@ -42,15 +45,12 @@ int inference_count = 0;
 
 int main(int argc, char* argv[]) {
```

Comment:
Or it could be moved to `testing/oak_tensorflow_hello_world`
The testing directory is used for both testing and debugging, and since the `hello_world` example is only used for debugging and doesn't appear in production, it can just be moved to the `testing/`

Reply:
I'm ok with moving apps to `testing`. Given those apps i.e. hello_world, gboard, etc are building tflite_micro, I'd prefer to putting them at `testing/tflite_micro` to have the same directory name `tflite_micro`. We'll put all sample, testing and debugging binaries in testing.